packages/app/cypress/e2e/runner/support/verify-failures.ts

```diff
@@ -155,51 +158,41 @@ export const verifyFailure = (options) => {
 
   if (verifyOpenInIde) {
     verifyIdeOpen({
-      file,
+      fileName,
       hasPreferredIde,
       action: () => {
-        cy.get('@Root').contains('.test-err-code-frame .runnable-err-file-path a', file)
+        cy.get('@Root').contains('.test-err-code-frame .runnable-err-file-path a', fileName)
         .click()
       },
     })
   }
 }
 
-const createVerifyTest = (modifier?: string) => {
-  return (title: string, opts: any, props?: any) => {
-    if (!props) {
-      props = opts
-      opts = null
-    }
-
-    props.specTitle ||= title
-
-    const verifyFn = props.verifyFn || verifyFailure.bind(null, props)
+export const createVerify = ({ fileName, hasPreferredIde }) => {
+  return (specTitle: string, props?: any) => {
+    props.specTitle ||= specTitle
+    props.fileName ||= fileName
+    props.hasPreferredIde = hasPreferredIde
```

Comment:
`props` doesn't look optional here if we're setting properties on it.

Reply:
Very true, I'll update